pysnn/learning.py

```diff
@@ -109,4 +108,5 @@ def update_eligibility_trace(self):
 
     def forward(self, reward):
         # TODO: add weight clamping?
+        self.update_eligibility_trace()
```

Comment:
Would prefer to call self.update_eligibility_trace() outside of the forward function for MSTDPET. The reward signal is likely not defined every timestep (or user should have the option not to), whereas the eligibility_trace should be updated after every timestep no matter the reward. Responsibility of the user to call it in their own network forward call.

Reply:
Makes sense!